staging/src/k8s.io/cli-runtime/pkg/genericclioptions/print_flags.go

```diff
@@ -81,7 +81,7 @@ func (f *PrintFlags) AllowedFormats() []string {
 	return ret
 }
 
-func (f *PrintFlags) ToPrinter() (printers.ResourcePrinter, error) {
+func (f *PrintFlags) UsedOutputFormat() string {
```

Comment:
Don't change this.

Reply:
Ok. I extracted this method to ensure that the resolved output format was the same. But I'll update the PR according to your comment, thanks.